codespell_lib/data/dictionary.txt

```diff
@@ -6204,14 +6210,18 @@ feeded->fed
 feetur->feature
 feeture->feature
 feld->field
+felisatus->felicitous
 femminist->feminist
 fempto->femto
+feonsay->fianc&eacute;e
```

Comment:
remove

or could maybe be unicode if we use unicode elsewhere

Reply:
Thanks. Fixed.